seed/seed20.go

```diff
@@ -496,34 +628,26 @@ func (s *seed20) loadEssentialMeta(filterEssential func(*asserts.ModelSnap) bool
 	essSnaps := model.EssentialSnaps()
 	const essential = true
 
+	// create channels
+	if filterEssential != nil {
+		// LoadEssentialMeta
+		s.snapsToConsiderCh = make(chan snapToConsider, 4)
```

Comment:
I would move that removed comment `// 4 = snapd+base+kernel+gadget` here

Reply:
I changed the code to use len instead